ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/newsletter/clientlibs/less/newsletter.less

```diff
@@ -35,4 +35,4 @@
             }
         }
     }
-}
\ No newline at end of file
+}
```

Comment:
why we are adding and delete the braces here

Reply:
only added margin-top.